source/docs/software/wpilib-tools/path-planning/wpilib-trajectories/examples/trajectorygeneration-2/java/ExampleTrajectory.java

```diff
@@ -0,0 +1,37 @@
+import java.util.ArrayList;
+
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.geometry.Translation2d;
+import edu.wpi.first.wpilibj.trajectory.TrajectoryGenerator;
+import edu.wpi.first.wpilibj.util.Units;
+
+class ExampleTrajectory {
+  public void generateTrajectory() {
+    final double startVelocity = 0;
+    final double endVelocity = 0;
+    final double maxVelocity = Units.feetToMeters(12.0);
+    final double maxAccel = Units.feetToMeters(12);
+
+    // 2018 cross scale auto waypoints.
+    var sideStart = new Pose2d(Units.feetToMeters(1.54), Units.feetToMeters(23.23),
```

Comment:
Don't use var

Reply:
It is very clear what type of object is being created here, saying `Pose2d sideStart = new Pose2d(...)` would be redundant. `var` should be used for situations like these.